yaki_database/create_tables.sql

```diff
@@ -0,0 +1,106 @@
+-- Begin with initiating the sequence for IDs regarding the captains, team, team mates, locations and declarations
+
+-- SEQUENCE: public.captain_id_seq
+-- DROP SEQUENCE IF EXISTS public.captain_id_seq;
+
+
+-- captain IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.captain_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.captain_id_seq
+-- will have to update the OWNER
+    OWNER TO postgres;
+
+-- team IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.team_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.captain_id_seq
+    OWNER TO postgres;
+
+-- team mate IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.team_mate_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.captain_id_seq
+    OWNER TO postgres;  
```

Comment:
Not sure to keep this instruction in the public scripts. If so, add create user script also